polkadot/runtime/common/src/impls.rs

```diff
@@ -150,6 +151,27 @@ impl TryConvert<&VersionedLocation, xcm::latest::Location> for VersionedLocation
 	}
 }
 
+/// Adapter for [`Contains`] trait to match [`VersionedLocatableAsset`] type converted to the latest
+/// version of itself where it's location matched by `L` and it's asset id by `A` parameter types.
+pub struct ContainsLatest<L, A>(PhantomData<(L, A)>);
+impl<L, A> Contains<VersionedLocatableAsset> for ContainsLatest<L, A>
+where
+	L: Contains<xcm::v4::Location>,
+	A: Contains<xcm::v4::Location>,
```

Comment:
as name `ContainsLatest` says :)
```suggestion
	L: Contains<xcm::latest::prelude::Location>,
	A: Contains<xcm::latest::prelude::Location>,
```